hack/tools/logcheck/main.go

```diff
@@ -110,3 +119,40 @@ func isUnstructured(fName string) bool {
 
 	return false
 }
+
+// isKeysValid check if all keys in keyAndValues is string type
+func isKeysValid(keyValues []ast.Expr, fun ast.Expr, pass *analysis.Pass, funName string) {
+	if len(keyValues)%2 != 0 {
+		pass.Report(analysis.Diagnostic{
+			Pos:     fun.Pos(),
+			Message: fmt.Sprintf("Invalid Number of arguments for %s", funName),
```

Comment:
This static analysis will be used for static analysis of every K/K PR, which means that almost every contributor will at some point need to fix errors from it. This puts requirement that descriptions for errors should be clear enough that someone without any knowledge of Structured Logging should be able to understand it.

Instead of message `nvalid Number of arguments for %s`, we should to with something more actionable:
For example: 
```
Uneven number of positional arguments. Additional arguments to InfoS should always be Key Value pairs.
Please check if there is any key or value missing.
```